build.sh

```diff
@@ -30,7 +30,8 @@ usage() {
 }
 
 SDKVER="2024-02-18_732134932b"
-
+DASHARO_SDK=${DASHARO_SDK:-"ghcr.io/dasharo/dasharo-sdk:latest"}
```

Comment:
Should point to a stable tag?

Reply:
So I assume we can merge and make v1.6.0 release https://github.com/Dasharo/dasharo-sdk/pull/11 ?